### PR TITLE
Use cl-lib macros instead of cl.el

### PR DESCRIPTION
**Pull request by syohex**
And fix arguments of error callback function

### CODE REVIEW

**Comment by etu** [comment]
Is there any big benefits of using cl-lib?

**Comment by syohex** [comment]
>  Your code should only ever load the main cl-lib file, which will load the others as needed. 

[Emacs document](https://www.gnu.org/software/emacs/manual/cl.html) says as above. cl.el functions and macros are alias of cl-lib functions and macros since Emacs 24.3. Original functions/macros should be used rather than alias version.

File: webpaste.el [hunk 1]
```diff
@@ -77,9 +78,9 @@ each run.")
                                         (when data
```
Comment (etu) [inline webpaste.el]: Would be niced if you changed this function as well to cl-function when changing all the others :)

Comment (syohex) [inline webpaste.el]: Sorry I mistook. I'll fixed.

**Comment by syohex** [comment]
Thanks for review. I've fixed and squashed commits.

**Comment by etu** [comment]
Thanks, merged. Melpa build is runing atm and this package is built right at the end so I think I got it in just in time for this build :)

**Pull request merged**